docker-compose/traefik/config/traefik.yaml

```diff
@@ -53,6 +53,7 @@ certificatesResolvers:
 providers:
   docker:
     exposedByDefault: false  # <-- (Optional) Change this to true if you want to expose all services
+  network: "frontend" # <-- Specify discovery network - This ensures correct name resolving and possible issues with containers, that are in multiple networks. E.g. Database container in a seperate network and a container in the frontend and database network.
```

Comment:
I assume you are referring to the optional [`network`](https://doc.traefik.io/traefik/providers/docker/#network) setting of the Docker provider here? If so, the indentation in the YAML is wrong by missing two spaces.
The double quotes around `frontend` aren't required here.